src/components/SelectedCounter/SelectedCounter.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './SelectedCounter.css';
+
+const SelectedCounter = ({ count, text }) => (
+  <div className="SelectedCounter__container">
```

Comment:
love the usage of BEM selector, I would call it simply `SelectedCounter` though, since double underscore in BEM syntax refers to an [E]lement, this is actually the [B]lock

Reply:
I wasn't aware of the BEM selector, I just thought it was a convention. Thanks for the heads up, I'll change it. 😅

Reply:
😄 I found this article useful [
MindBEMding – getting your head ’round BEM syntax](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/) when I first started with BEM convention.